src/structures/Player.ts

```diff
@@ -0,0 +1,301 @@
+// tslint:disable: member-ordering max-line-length
+import { Structure, State } from "./Utils";
+import { Manager } from "./Manager";
+import { Queue } from "./Queue";
+import { Node } from "./Node";
+
+/** The PlayerOptions interface. */
+export interface PlayerOptions {
+    /** The guild the Player belongs to. */
+    guild: any;
+    /** The text channel the Player belongs to. */
+    textChannel: any;
+    /** The voice channel the Player belongs to. */
+    voiceChannel?: any;
+    /** The node the Player uses. */
+    node?: string;
+    /** The initial volume the Player will use. */
+    volume?: number;
+    /** If the player should mute itself. */
+    selfMute?: boolean;
+    /** If the player should deafen itself. */
+    selfDeafen?: boolean;
+}
+
+/** The Track interface. */
+export interface Track {
+    /** The base64 encoded track. */
+    readonly track: string;
+    /** The title of the track. */
+    readonly title: string;
+    /** The identifier of the track. */
+    readonly identifier: string;
+    /** The author of the track. */
+    readonly author: string;
+    /** The length of the track. */
+    readonly length: number;
+    /** If the track is seekable. */
+    readonly isSeekable: boolean;
+    /** If the track is a stream.. */
+    readonly isStream: boolean;
+    /** The uri of the track. */
+    readonly uri: string;
+    /** The thumbnail of the track. */
+    readonly thumbnail: string;
+    /** The user that requested the track. */
+    readonly requester: any;
+    /** Displays the track thumbnail with a size in "0", "1", "2", "3", "default", "mqdefault", "hqdefault", "maxresdefault". Only for youtube as others require an API. */
+    displayThumbnail(size?: string): string;
```

Comment:
why string ? why not a separate type here ? 
```ts
export type sizes = "0" | "1" | "2" | "3" | "default" | "mqdefault" | "hqdefault" | "maxresdefault";
displayThumbnail(size?: sizes): string;
```

Reply:
Because I did that at about 4 in the morning and didn't think about making it a type. And yeah I'll change it to that.